### PR TITLE
Issue #13109: Kill mutation for CustomImportOrderCheck

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #13109: Kill mutation for CustomImportOrderCheck

--------

Check :- 
https://checkstyle.org/config_imports.html#CustomImportOrder

-----

# Mutation Covered
https://github.com/checkstyle/checkstyle/blob/3f77dd11d7e7ac8e23ecc817a1caeeea7d1a489d/config/pitest-suppressions/pitest-imports-suppressions.xml#L2-L19

-----------

# Explaination
Test cases added

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSpecialImportsRegExp.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+standardPackageRegExp = junit
+thirdPartyPackageRegExp = (default).*
+specialImportsRegExp = (default)^$
+separateLineBetweenGroups = (default)true
+sortImportsInGroupAlphabetically = (default)false
+
+
+*/
+
+//non-compiled with javac: special package and requires imports from the same package
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSpecialImportsRegExp.java:13]: Remove this comment, ilit is compile able.

**Review by romani — CHANGES_REQUESTED** [review]
Items

**Comment by romani** [comment]
Please pull latest our master, semaphore is fixed

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSpecialImportsRegExp.java (hunk 1 above)
Comment (Kevin222004) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSpecialImportsRegExp.java]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**